### DISCUSSION TITLE
Clicking on any link in the blog sidebar, makes it scroll back to the top

### DISCUSSION THREAD

**Discussion by laurent22** [facebook/docusaurus#9466] [Q&A] [answered]
Using Docusaurus 2.4.3, whenever a link is clicked in the **Blog** sidebar, the sidebar scrolls back up to the top. I don't see this behaviour in the regular ("Docs") sidebar though. Any idea if there's any way to fix this?

**Comment by slorber** [comment, marked as answer]
Afaik it's not possible because blog post routes do not share a parent layout that includes the sidebar. It would require refactoring the theme a bit to support this

**Reply by ruanqizhen** [reply]
Are there any updates on this?

**Reply by slorber** [reply]
This is something that I'd like to fix asap because it's also annoying me. 
If the theme refactor needs a breaking change, we'll have to postpone this to V4.


Not exactly the same, but slightly related to another problem we have: https://github.com/facebook/docusaurus/issues/2891

**Reply by ruanqizhen** [reply]
Is it possible to scroll the sidebar to the highlighted item after a refresh? It may not be as good as preserving the state, but it's still better than scrolling to the top.
